tests/test_env.py

```diff
@@ -5,7 +5,7 @@
 from plumbum import local
 from plumbum._testtools import skip_on_windows
 
-with contextlib.suppress(ModuleNotFoundError):
+with contextlib.suppress(ModuleNotFoundError, ImportError):
```

Comment:
```suggestion
with contextlib.suppress(ImportError):
```

`ModuleNotFoundError` is an `ImportError`

Not sure about the other change yet, just noting this quickly.